### ISSUE TITLE
download from file:/// does not show 'Not Secure' - follow up to 1269

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#1590]
## Description 
Found while testing https://github.com/brave/brave-browser/issues/1269

Scenario 3 from test plan on https://github.com/brave/brave-core/pull/597 does not show the unlock icon/'Not Secure' on hover when downloading from text file.

## Steps to Reproduce

Download from a file://:
1. Drag and drop a text file onto the browser.
2. Right-click on the content and choose "Save as...".
3. Select destination if asked.
4. Observe download item appear in the download shelf on the bottom of the browser. --> Download item does not show the unlocked lock icon but the origin URL is file:///
5. Point mouse to the download item and wait for the tool tip to appear. --> Tool tip does not contain the words "Not secure"

## Actual result:
Download item missing unlock icon:
![screen shot 2018-10-15 at 4 02 35 pm](https://user-images.githubusercontent.com/28145373/46975102-bf1e3d00-d093-11e8-9171-4b60cea524e4.png)

Hover tool tip does not contain 'Not Secure':
![screen shot 2018-10-15 at 4 03 03 pm](https://user-images.githubusercontent.com/28145373/46975134-cfceb300-d093-11e8-8c8e-7c4f8b975a24.png)

## Expected result:
Unlock icon on download item and hover tool tip with 'Not Secure' text

## Reproduces how often: 
easily

## Brave version (chrome://version info)

Brave | 0.55.14 Chromium: 70.0.3538.54 (Official Build) beta(64-bit)
-- | --
Revision | 4f8e578b6680574714e9ed3bb9f02922b4dde40d-refs/branch-heads/3538@{#937}
OS | Mac OS X

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? yes reproduced on beta
- Does it reproduce on browser-laptop? n/a

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional Information
cc @diracdeltas

**Comment by diracdeltas** [comment]
this is actually expected behavior; file:/// is considered a secure origin. cc @mkarolin

**Comment by srirambv** [comment]
Muon showed the same but was late renamed to `Local File`. https://github.com/brave/browser-laptop/pull/8786. Should probably do the same here as well.

**Comment by mkarolin** [comment]
Yes, sorry, in the original implementation file:/// wasn't considered a secure source, but then on @diracdeltas' suggestion I switched to using an API function that's more sophisticated than just checking for `https`. `file:///` is considered a secure source by that API (as well as `about:` and even `http://localhost`, if you ever test those).

**Comment by LaurenWags** [comment]
@diracdeltas @mkarolin  should scenario 3 on test plan in https://github.com/brave/brave-core/pull/597 be updated?

**Comment by mkarolin** [comment]
@LaurenWags yes, I'll update.

**Comment by mkarolin** [comment]
Updated.
